.github/file-filters.yml

```
@@ -0,0 +1,4 @@
+# This is used by the action https://github.com/dorny/paths-filter
+
+high_risk_code: &high_risk_code
+  - '.github/file-filters.yml'
```

I also agree that the Native references on TypeScript are good candidates for it, what do you think @krystofwoldrich?
```suggestion
  - '.github/file-filters.yml'
  - 'src/js/vendor/react-native/index.ts'
  - 'src/js/touchevents.tsx'
  - 'src/js/utils/rnlibraries.ts'
  - 'src/js/utils/rnlibrariesinterface.ts'
  - 'src/js/NativeRNSentry.ts'
```